frontend/src/metabase/parameters/components/ValuesSourceSettings/ValuesSourceSettings.unit.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import userEvent from "@testing-library/user-event";
+import { Parameter, ValuesQueryType } from "metabase-types/api";
+import { createMockParameter } from "metabase-types/api/mocks";
+import { renderWithProviders, screen } from "__support__/ui";
+import ValuesSourceSettings from "./ValuesSourceSettings";
+
+interface SetupOpts {
+  parameter: Parameter;
+}
+
+const setup = ({ parameter }: SetupOpts) => {
+  const onChangeQueryType = jest.fn();
+  const onChangeSourceSettings = jest.fn();
+
+  renderWithProviders(
+    <ValuesSourceSettings
+      parameter={parameter}
+      onChangeQueryType={onChangeQueryType}
+      onChangeSourceSettings={onChangeSourceSettings}
+    />,
+  );
+
+  return { onChangeQueryType, onChangeSourceSettings };
+};
+
+describe("ValuesSourceSettings", () => {
+  it.each<ValuesQueryType>(["list", "search"])(
+    "should allow changing values settings for %i",
```

Comment:
`%i` [is supposedly for integer](https://jestjs.io/docs/api#1-testeachtablename-fn-timeout) so the test name doesn't render correctly.

<img width="593" alt="image" src="https://user-images.githubusercontent.com/1937582/215107549-7f251123-559f-4a08-b209-9327329b2a4e.png">

Changing to `%s` fixes the issue.

```suggestion
    "should allow changing values settings for %s",
```

<img width="613" alt="image" src="https://user-images.githubusercontent.com/1937582/215107655-42d5f7da-fd69-4465-818e-7c2b289dcb97.png">